fathom/src/main.rs

```diff
@@ -163,8 +164,13 @@ fn main() -> ! {
                 Input::File(path) => driver.read_source_path(&path),
             };
 
-            let mut reader = BufReader::new(std::fs::File::open(binary_path).unwrap()); // TODO: report errors
-            let status = driver.read_format(file_id, &mut reader);
+            // TODO: report errors
+            let mut file = std::fs::File::open(binary_path).unwrap();
+            let mut data = Vec::new();
+            file.read_to_end(&mut data).unwrap();
+            let buffer = fathom::core::binary::Buffer::from(data.as_slice());
```

Comment:
Need to deal with these errors better. Maybe I should move the binary file processing into the driver 🤔